### ISSUE TITLE
Support adding tags

### ISSUE THREAD

**Issue by hhzl** [ewnd9/anki-apkg-export-app#3]
A suggestion:

Support adding tags, for example with the following syntax

    # tag:  tagA
    contentOfFrontOfCard1  - contentOfBackOfCard1
    contentOfFrontOfCard2  - contentOfBackOfCard2
    contentOfFrontOfCard3  - contentOfBackOfCard3

    # tag:  tagB
    contentOfFrontOfCard4  - contentOfBackOfCard4
    contentOfFrontOfCard5  - contentOfBackOfCard5

Result:
The cards 1, 2 and 3 have 'tagA' and cards 4 and 5 have 'tagB'.   

Another possibility, no hash sign. It is unlikely that the content of a card starts with 'tag:'

    tag:  tagA
    contentOfFrontOfCard1  - contentOfBackOfCard1
    contentOfFrontOfCard2  - contentOfBackOfCard2
    contentOfFrontOfCard3  - contentOfBackOfCard3

    tag:  tagB
    contentOfFrontOfCard4  - contentOfBackOfCard4
    contentOfFrontOfCard5  - contentOfBackOfCard5

And then in addition there could be support for several tags

    tags:  tagB  tagC
    contentOfFrontOfCard6  - contentOfBackOfCard6
    contentOfFrontOfCard7  - contentOfBackOfCard7

A argument for using the hash sign 

        # tag:  tagA
is that the text formats nicely as markdown.

**Comment by ewnd9** [comment]
Yep, hash looks good. I built it as example and don't use it myself, so pull request welcome.

---

By the way, I moved all repos to one org and started documentation in https://github.com/repeat-space/srs-knowledge

Would you like to contribute your notes from https://github.com/hhzl/LearnWords2/blob/master/docs/Developer_notes.md and join efforts?

**Comment by hhzl** [comment]
OK, Let's go for hash. I understand that I'll have to do a pull-request.

Yes, contributing my notes and thus joining efforts is fine.

Priorities for me are
1. Documentation of Anki apkg file content. This includes the SQLite database tables with examples of entries.
2. Documentation how to move content in and out of Anki apkg files.
